python/src/spark_rapids_ml/classification.py

```diff
@@ -1013,6 +1069,9 @@ def intercept(self) -> float:
     def coefficientMatrix(self) -> Matrix:
         """
         Model coefficients.
+        Note Spark CPU uses denseCoefficientMatrix.compressed that may return a sparse vector
```

Comment:
Can this be addressed using cpu() method?  Convert to cpu and invoke coefficientMatrix on cpu model.   Might need to extend cpu method to multi-class case.

Reply:
Spark CPU fit uses coefficientMatrix.compressed right after training to create LogisticRegressionModel. cpu() method does not call fit so seems unlikely to help. 

It it feasible to implement the coefficientMatrix.compressed function in python, after we know how to calculate the sizes of DenseVector and SparseVector. 

Created a Jira ticket 775 to track this.